### ISSUE TITLE
Can VSCode Stay in background?

### ISSUE THREAD

**Issue by KaneGreen** [Microsoft/vscode#46480]
### TL;DR
I'm **requesting a new feature**:  Stay in background.  
An new option about the exit behavior of the close button:
* **close the window** directly and exit
* **close the files or folder** and **minimize** the windows to task bar

----
## Full version
I think VSCode start slowly. So I'm thinking can VSCode add an option in the settings that user can determine the exit behavior that user clicked the close button: some users prefer to **close the window directly and exit**, just like now VSCode dose, while maybe some other users like me prefer to **close the files or folder** and **minimize the windows to task bar**, not exit.  
This way, when back to VSCode, it "starts" fast.  

Maybe staying in background need take up some memory, but I think there do exist people using large memory and don't care it. And I think it's worthy. And people have right to choose.
